src/ITILFollowup.php

```diff
@@ -365,6 +365,13 @@ public function prepareInputForAdd($input)
             );
         }
 
+        // _do_not_compute_status can be passed from rule
+        // if isset it will be used to set _do_not_compute_takeintoaccount
+        // and prevent take into account delay computation
+        if (isset($input['_do_not_compute_status'])) {
+            $input['_do_not_compute_takeintoaccount'] = $input['_do_not_compute_status'];
+        }
+
```

Comment:
There is a dedicated action that can be used for that in the rules.
![image](https://github.com/user-attachments/assets/01b72a7e-1d49-480f-af32-f2fadc158abc)